gallery/visualtests/nxCombobox.js

```diff
@@ -0,0 +1,145 @@
+      const inputSelector = `${basicExampleSelector} .nx-combobox__input input`,
+          dropdownButtonSelector = `${basicExampleSelector} .nx-dropdown-button`,
+          [component, input] = await waitAndGetElements(basicExampleSelector, inputSelector);
+
+      await input.focus();
+      await getPage().keyboard.type('loo');
+      await getPage().waitForSelector(dropdownButtonSelector);
+      await checkScreenshot(component, 800, 88);
+    });
+  });
+
+  describe('when displaying empty message', function() {
+    it('looks right', async function() {
+      const inputSelector = `${basicExampleSelector} .nx-combobox__input input`,
+          emptyMessageSelector = `${basicExampleSelector} .nx-combobox__empty-message`,
+          [component, input] = await waitAndGetElements(basicExampleSelector, inputSelector);
+
+      await input.focus();
+      await getPage().keyboard.type('q');
+      await getPage().waitForSelector(emptyMessageSelector);
+      await checkScreenshot(component, 800, 88);
+    });
+  });
+
+  describe('when displaying an error', function() {
+    it('looks right', async function() {
+      const inputSelector = `${errorExampleSelector} .nx-combobox__input input`,
+          errorSelector = `${errorExampleSelector} .nx-alert--load-error`,
+          [component, input] = await waitAndGetElements(errorExampleSelector, inputSelector);
+
+      await input.focus();
+      await getPage().keyboard.type('1');
+      await getPage().waitForSelector(errorSelector);
+      await checkScreenshot(component, 300, 195);
+    });
+  });
+
+  describe('when disabled', function() {
+    it('looks right', simpleTest(disabledExampleSelector));
+  });
+
+  describe('close the dropdown when not focused', function() {
+    it('looks right', async function() {
+      const inputSelector = `${basicExampleSelector} .nx-combobox__input input`,
+          dropdownButtonSelector = `${basicExampleSelector} .nx-dropdown-button`,
+          [component, input] = await waitAndGetElements(basicExampleSelector, inputSelector);
+
+      await input.focus();
+      await getPage().keyboard.type('a');
+      await getPage().waitForSelector(dropdownButtonSelector);
+      await blurElement(input);
+      await checkScreenshot(component, 800, 55);
+    });
+  });
+
+  describe('dropdown button has a blue background when selected', function() {
+    it('looks right', async function() {
+      const inputSelector = `${basicExampleSelector} .nx-combobox__input input`,
+          dropdownButtonSelector = `${basicExampleSelector} .nx-dropdown-button`,
+          [component, input] = await waitAndGetElements(basicExampleSelector, inputSelector);
+
+      await input.focus();
+      await getPage().keyboard.type('a');
+      await getPage().waitForSelector(dropdownButtonSelector);
+      await getPage().keyboard.press('ArrowDown');
+      await checkScreenshot(component, 800, 376);
+    });
+  });
+
+  it('passes a11y checks', a11yTest());
```

Comment:
We need to check a11y on a bunch of different states.  See the `NxSearchDropdown` a11y tests.

Reply:
Updated here ffadf20856e39d93aec08617f531230281525995.

Reply:
I didn't realize a11yTest does nothing different if no `beforeEach` set up within `describe` block, thanks for letting me know! Updated here f77ac497df5b35bf941474d35188f817eccd8f78.